src/pages/Reports/state/OverviewMapState.js

```diff
@@ -38,9 +58,13 @@ actions.setSelectedReport = (selectedReport, zoomIn) => async (
   dispatch,
   getState
 ) => {
-  const { reports } = getState().ReportsState.OverviewMapState;
+  const {
+    reports,
+    reportFetchState
+  } = getState().ReportsState.OverviewMapState;
 
-  if (!reports.length) {
+  if (!reports.length && reportFetchState !== 'pending') {
+    logger(reportFetchState);
     await loadReportsThunk(dispatch);
   }
```

Comment:
This block caused an infite loop by both calling and being called by `loadReportsThunk`.